src/hosted-buttons/utils.js

```diff
@@ -149,19 +174,14 @@ export const buildHostedButtonOnApprove = ({
           context_id: data.orderID,
         }),
       }).then((response) => {
+        // remove the popup fallback message, if present
+        document.querySelector(`.${popupFallbackClassName}`)?.remove();
```

Comment:
this is another weird thing in this PR. I was really hoping to have all the dom manipulation contained within the popup handling code, but we need this here to handle a specific use case:
1. buyer uses inline guest to checkout, has popup blocked, error message is rendered in the dom
2. buyer uses a different payment method ("paypal") and when they click on the  button, there should no longer be an error message.